### PR TITLE
updating rest of templates, cleaning up duplicates

### PR DESCRIPTION
**Pull request by olsenme**
@skayani  @serenaz  I removed the duplicate folder as Jenny seems to be using the correct one. I also fixed the other multiple resource group templates. @skayani  please test out before merge as I left the default values in there and found a minor syntax error in the other template. Thanks

### CODE REVIEW

File: templates/delegated-resource-management-eligible-authorizations/subscription/test.ps1 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Connect-AzAccount
```
Comment (skayani) [inline templates/delegated-resource-management-eligible-authorizations/subscription/test.ps1:1]: This file isn't needed?

File: templates/delegated-resource-management-eligible-authorizations/subscription/subscription-managing-tenant-approvers.json [hunk 2]
```diff
@@ -7,21 +7,21 @@
             "metadata": {
                 "description": "Specify a unique name for your offer"
             },
-            "defaultValue": "<to be filled out by MSP> Specify a title for your offer"
+            "defaultValue": "Relecloud Premium Managed Services"
```
Comment (skayani) [inline templates/delegated-resource-management-eligible-authorizations/subscription/subscription-managing-tenant-approvers.json:10]: I think this is a good opportunity to remove the defaultValues going forward. Please refer to [my comment](https://github.com/Azure/Azure-Lighthouse-samples/pull/112#issuecomment-892084547) regarding specifying defaultValues in the PR I merged.

Comment (serenaz) [reply templates/delegated-resource-management-eligible-authorizations/subscription/subscription-managing-tenant-approvers.json:10]: I agree. Also need to remove PII for the tenant/principal ids

File: templates/delegated-resource-management-eligible-authorizations/subscription/test.ps1 (hunk 1 above)
Comment (olsenme) [reply templates/delegated-resource-management-eligible-authorizations/subscription/test.ps1:1]: ok I fixed this

File: templates/delegated-resource-management-eligible-authorizations/subscription/subscription-managing-tenant-approvers.json (hunk 2 above)
Comment (olsenme) [reply templates/delegated-resource-management-eligible-authorizations/subscription/subscription-managing-tenant-approvers.json]: ok I addressed this in the 2nd commit

**Comment by serenaz** [comment]
thank you both for reviewing :)

**Pull request merged**